modules/apps/object/object-storage-salesforce/src/main/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/SalesforceObjectEntryManagerImpl.java

```diff
@@ -171,6 +171,24 @@ public String getStorageType() {
 		return ObjectDefinitionConstants.STORAGE_TYPE_SALESFORCE;
 	}
 
+	@Override
+	public ObjectEntry partialUpdateObjectEntry(
+			long companyId, DTOConverterContext dtoConverterContext,
+			String externalReferenceCode, ObjectDefinition objectDefinition,
+			ObjectEntry objectEntry, String scopeKey)
+		throws Exception {
+
+		ObjectEntry existingObjectEntry = getObjectEntry(
+			companyId, dtoConverterContext, externalReferenceCode,
+			objectDefinition, scopeKey);
+
+		preparePatch(objectEntry, existingObjectEntry);
```

Comment:
@pedro-oliveira446 I also think we can get this method off, for example:

```java
	public default ObjectEntry partialUpdateObjectEntry(
			long companyId, DTOConverterContext dtoConverterContext,
			String externalReferenceCode, ObjectDefinition objectDefinition,
			ObjectEntry objectEntry, String scopeKey)
		throws Exception {

		ObjectEntry existingObjectEntry = getObjectEntry(
			companyId, dtoConverterContext, externalReferenceCode,
			objectDefinition, scopeKey);

		if (objectEntry.getDateCreated() != null) {
			existingObjectEntry.setDateCreated(objectEntry.getDateCreated());
		}

		if (objectEntry.getDateModified() != null) {
			existingObjectEntry.setDateModified(objectEntry.getDateModified());
		}

		if (objectEntry.getKeywords() != null) {
			existingObjectEntry.setKeywords(objectEntry.getKeywords());
		}

		if (objectEntry.getProperties() != null) {
			Map<String, Object> properties =
				existingObjectEntry.getProperties();

			properties.putAll(objectEntry.getProperties());

			existingObjectEntry.setProperties(properties);
		}

		if (objectEntry.getTaxonomyCategoryIds() != null) {
			existingObjectEntry.setTaxonomyCategoryIds(
				objectEntry.getTaxonomyCategoryIds());
		}

		return updateObjectEntry(
			companyId, dtoConverterContext, externalReferenceCode,
			objectDefinition, existingObjectEntry, scopeKey);
	}
```

I know that the method preparePatch is used in DefaultObjectEntryManagerImpl side, there we can also duplicate my goal here is keep the interface clear.